examples/query/react/basic/src/test/server/index.ts

```diff
@@ -0,0 +1,7 @@
+import { setupServer } from 'msw/node'
+import { handlers } from './serverHandlers'
+
+const server = setupServer(...handlers)
+
+export * from 'msw'
+export { server }
```

Comment:
Would y'all want the server and the other `msw` methods exported here, or from `setupTests.ts`? Personally I think exporting it from here is nice because then in the test file the import is clear:
```typescript
import { server } from './test/server'
```

Reply:
Looks good from here.